server/game/provincecard.js

```diff
@@ -1,9 +1,9 @@
 const _ = require('underscore');
 
-const BaseCard = require('./basecard');
+const DrawCard = require('./drawcard');
 const { Locations, EffectNames } = require('./Constants');
 
-class ProvinceCard extends BaseCard {
+class ProvinceCard extends DrawCard {
```

Comment:
Why does ProvinceCard need to inherit from DrawCard?  This feels like a fairly major change...